charts/nginz/templates/conf/_nginx.conf.tpl

```diff
@@ -125,7 +125,9 @@ http {
 
   map $http_origin $cors_header {
       default "";
-      "~^https://([^/]+\.)?{{ .Values.nginx_conf.external_env_domain | replace "." "\\." }}(:[0-9]{2,5})?$" "$http_origin";
+    {{ range $origin := .Values.nginx_conf.allowlisted_origins }}
+      "https://{{ $origin }}.{{ .Values.nginx_conf.external_env_domain}} "$http_origin";
```

Comment:
```suggestion
      "https://{{ $origin }}.{{ .Values.nginx_conf.external_env_domain}}" "$http_origin";
```

Reply:
116988c  (i also kept the `$` from #2073)

Reply:
Sorry for that :/